composeApp/build.gradle.kts

```diff
@@ -54,6 +54,13 @@ kotlin {
             implementation(libs.ktor.client.core)
             implementation(libs.kaml)
         }
+
+        commonTest.dependencies {
+            implementation(kotlin("test"))
```

Comment:
Replace with kotest.

Reply:
Not that easy with desktopTest to make the test discoverable. Let's stay with kotlin.test for now.